image_provisioner/playbooks/roles/openshift-package-install/tasks/main.yaml

```diff
@@ -1,6 +1,9 @@
 ---
-- name: install openshift packages
-  yum: name={{ item }} state=latest
+- name: install openshift packages with openvswitch from rhel7-fast-datapath-rpms by disabling rhel7-fast-datapath-htb repo 
+  yum: 
+    name: "{{ item }}"
+    state: latest
+    disablerepo: rhel7-fast-datapath-htb
```

Comment:
My concern is that we could still lose the package via yum update later (at runtime, not during provisioning).   But I guess that is out of our control since aos-ansible is mucking with the same repos.

Reply:
We would need to add the new staging repo Justin is sync-ing from now, as a task in the repo-install role